Resources/Prototypes/_Harmony/Catalog/Fills/Lockers/security.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  id: LockerBrigmedicFilledHarmony
+  suffix: Brigmedic, Filled, Harmony
+  parent: LockerBrigmedic
+  components:
+  - type: StorageFill
+    contents:
+      - id: TrackingImplanter
+        amount: 2
+      - id: ClothingOuterHardsuitBrigmedic
+      - id: ClothingOuterCoatAMG
+      - id: MedkitCombatFilled
+      - id: ClothingEyesGlassesBrigmedicHarmony
+      - id: LauncherSyringeHarmony
+      - id: MiniSyringe
+        amount: 2
+      - id: ClothingBackpackDuffelSurgeryFilled
+      - id: ClothingBeltBrigmedHarmony
```

Missing newline at the end of file.
```suggestion
      - id: ClothingBeltBrigmedHarmony

```